tests/testsuite/namespace.go

```diff
@@ -249,6 +251,28 @@ func CleanNamespaces() {
 	}
 }
 
+// waitForNamespaceSCCAnnotations waits up to 30s for the cluster-policy-controller to add the SCC related
+// annotations to the provided namespace.
+func waitForNamespaceSCCAnnotations(c kubecli.KubevirtClient, nsName string) {
+	EventuallyWithOffset(1, func() error {
+		ns, err := c.CoreV1().Namespaces().Get(context.Background(), nsName, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		if ns.Annotations == nil {
+			return fmt.Errorf("namespace %q annotaions are nil", nsName)
+		}
```

Comment:
No need a special case for nil, because the for will just range on nothing anyway.

I also don't think we need to have that specific error that the annotations are nil
you can also just print the annotations instead if it is important for you.